build.rs

```diff
@@ -28,7 +28,5 @@ fn main() {
 
     perf_bindings
         .write_to_file(out_path.join("perf_event.rs"))
-        .expect(
-        "Unable to write perf_event bindings to ./src/bindings/perf_event.rs",
-    );
+        .expect("Unable to write perf_event bindings to ./src/bindings/perf_event.rs");
```

Comment:
Can you run the `git hook` before each commit next time? That way this commit would just have the change to `src/test/mod.rs`